silo/tests/test_views.py

```diff
@@ -138,3 +147,74 @@ def test_index_get_unauthenticated_no_activity_url(self):
         request.user = AnonymousUser()
         with self.assertRaises(ImproperlyConfigured):
             views.IndexView.as_view()(request)
+
+
+class ExportViewsTest(TestCase, MongoTestCase):
+    def setUp(self):
+        factories.ReadType(read_type='CustomForm')
+        self.tola_user = factories.TolaUser()
+        self.factory = APIRequestFactory()
+        self.silo_id = None
+
+    def tearDown(self):
+        if self.silo_id:
+            # Have to remove the created lvs
+            lvss = LabelValueStore.objects.filter(silo_id=self.silo_id)
+            for lvs in lvss:
+                lvs.delete()
+
+    def test_export_csv(self):
+        self.tola_user.user.is_staff = True
+        self.tola_user.user.is_superuser = True
```

Comment:
Do we need to be superuser for this?

Reply:
Doesn't matter. I'm gonna create another test with a normal user.